src/components/Article.jsx

```diff
@@ -10,10 +14,11 @@ export const Article = ({ article }) => {
 				<div>
 					<div>{article.author}</div>
 					<div>Posted on: {date}</div>
-					<div>👍: {article.votes}</div>
 				</div>
 				<img src={article.article_img_url} alt="{article.title}" />
 				<p>{article.body}</p>
+				<div>👍: {article.votes}</div>
```

Comment:
if your Vote component is going to display the vote number anyway then it doesn't make sense to display it above too. It actually feels more confusing because when I vote the two numbers don't match up until I reload the whole page again.